src/TransCreator/calculator.js

```diff
@@ -0,0 +1,86 @@
+export const keyboardLayout = [
+  [1, 4, 7, 'C'],
+  [2, 5, 8, '0'],
+  [3, 6, 9, '.'],
+  ['+', '-', 'remove', 'Add'],
```

Comment:
The value name for `remove` and `Add`, please ensure the same format

Reply:
fixed